src/pages/SettingsPage.qml

```diff
@@ -124,6 +125,17 @@ Page {
 
                 onCheckedChanged: closeAllTabsConfig.value = checked
             }
+
+            TextSwitch {
+                //: Label for text switch that disables JavaScript globally for all tabs
+                //% "Disable JavaScript"
+                text: qsTrId("settings_browser-la-disable_javascript")
+                //% "On all newly loaded pages JavaScript will be disabled"
```

Comment:
Hmm... This is rather long sentence. Also this could have conditions as per WebEngineSettings.javascriptEnabled:

```
                description: WebEngineSettings.javascriptEnabled ?
                                     //% "Enabled (recommended)"
                                     qsTrId("settings_browser-la-enabled_javascript_description") :
                                     //% "Blocked, all pages may not load properly"
                                     qsTrId("settings_browser-la-disable_javascript_description")

```
Indentation are surely wrong in above. Point is that description should tell / describe what are the implications.

Reply:
Updated this